include/onnxruntime/core/framework/allocator.h

```diff
@@ -22,14 +22,14 @@ struct OrtAllocatorInfo {
   OrtMemType mem_type;
   OrtAllocatorType type;
 
-  constexpr OrtAllocatorInfo(const char* name1, OrtAllocatorType type, int id1 = 0, OrtMemType mem_type1 = OrtMemTypeDefault)
+  constexpr OrtAllocatorInfo(const char* name_, OrtAllocatorType type_, int id_ = 0, OrtMemType mem_type_ = OrtMemTypeDefault)
```

Comment:
>name_ [](start = 41, length = 5)

Isn't using a trailing '_' reserved for data members of classes? If you don't like the '1' suffix maybe '_in' would be better?

Reply:
google coding standard is saying, for struct, tailing "_" is used here, it's exactly the opposite way of naming in a class.

Reply:
it's not a big concern from me about naming, and I can revert it if you have preference :) with "_" prefix is not that good, I feel :)

btw, I'm not good at naming .....

Reply:
up to you. it's not a big deal. more about consistency. 

someone new to the code may see '_' being used as a suffix in parameters here and not realize that is because it's the ctor of a struct. slightly cleaner to me to never use  a '_' suffix for a parameter anywhere so there's never any chance (tiny as it may be in this case) of confusion.

---
In reply to: [255731657](https://github.com/Microsoft/onnxruntime/pull/467#discussion_r255731657) [](ancestors = 255731657)

Reply:
I didn't mean use '_' as a prefix though. meant 'name_in', 'id_in', etc.

---
In reply to: [255732616](https://github.com/Microsoft/onnxruntime/pull/467#discussion_r255732616) [](ancestors = 255732616,255731657)